### PR TITLE
DUI: move curve divide to action

### PR DESCRIPTION
**Pull request by aosyatnik**
Fix bug with Curve.Divide.
image

"Spacing Rule Layout" should be in Curve.Actions

Must be merged after [this PR](https://github.com/DynamoDS/Dynamo/pull/3869).

Reviewers
@Benglin , please take a look.

Link to YouTrack:
[MAGN-6528](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6528) DUI: Curve was considered as namespace on top of revit

### CODE REVIEW

**Comment by kronz** [comment]
@Benglin @aosyatnik @ikeough Hold on.  Spacing Rule Layout is a weird and Revit specific functionality that can only be used in conjunction with the DividedPath functionality.  

Probably only makes sense (barely) in Revit>Elements>DividedPath>Action



BTW
Why is Curve in it's own space, seperate out in the Geometry library?  That a known bug?

**Comment by elayabharath** [comment]
@kronz the curve was separated because 'Divide > Spacing Rule Layout' was added to the curve which then acts like a namespace. Once we move it out of the curve / force it to be Curve.Actions, the bug will be fixed

**Comment by aosyatnik** [comment]
@elayabharath , @kronz , so, what should I do? What is the appropriate category for "Spacing Rule Layout"?

**Comment by kronz** [comment]
@aosyatnik  Revit>Elements>DividedPath>Action

**Comment by aosyatnik** [comment]
Ok, changed to  Revit>Elements>DividedPath>Action. @Benglin , please, merge this PR after https://github.com/DynamoDS/Dynamo/pull/3869.

File: src/Libraries/RevitNodesUI/RevitDropDown.cs [hunk 1]
```diff
@@ -562,7 +562,7 @@ public StructuralColumnTypes()
     }
 
     [NodeName("Spacing Rule Layout")]
-    [NodeCategory(BuiltinNodeCategories.GEOMETRY_CURVE_DIVIDE)]
+    [NodeCategory("Revit.Elements.DividedPath.Actions")]
```
Comment (Benglin) [inline src/Libraries/RevitNodesUI/RevitDropDown.cs]: This is not what I meant, @aosyatnik. I would like you to make a new `NodeCategories.cs` in all three Revit branches. Then you can introduce `BuiltinNodeCategories.GEOMETRY_CURVE_DIVIDE` in that new file.

**Comment by aosyatnik** [comment]
Hi, @Benglin , I created NodeCategories.cs, which stores Revit categories.

**Comment by Benglin** [comment]
Thanks @aosyatnik, LGTM! I'm merging this in, can you please cross-merge these files to `Revit2015` and `Revit2016` as early as you can? Otherwise they will break, thanks!

**Pull request merged**